tests/acceptance/00_basics/macros/if_triple.cf

```diff
@@ -0,0 +1,48 @@
+######################################################
+#
+#  Test that @if works with patch level
+#
+#####################################################
+
+body common control
+{
+      inputs => { "../../default.cf.sub" };
+      bundlesequence  => { default("$(this.promise_filename)") };
+      version => "1.0";
+}
+
+bundle common test
+{
+@if minimum_version(3.7.100)
+  classes:
+      "expected" expression => "any";
+@endif
+
+@if minimum_version(3.8.0)
+  classes:
+      "expected2" expression => "any";
+@endif
+
+@if minimum_version(2.100.0)
+  classes:
+      "expected_2_100" expression => "any";
+@endif
+
+@if minimum_version(3.8.300)
```

Comment:
Heh, I have the feeling we'll have to change this every time we make a new feature release. But at the same time I can't think of a better way to test it either. :-)